tests/actions/__snapshots__/render.test.js.snap

```diff
@@ -8,7 +8,7 @@ Result {
 \\"Settings.PushPermissionsRequest.Subtitle.COPY\\" = \\"Stay up to date\\";
 \\"Something.With.An.Arbitrary.Amount.Of.Nesting.COPY\\" = \\"We support nesting as deep as you'd like\\";
 \\"Delivery.Widget.Title.COPY\\" = \\"Help: %1$@\\";
-\\"Delivery.Widget.SpecialCharacters.COPY\\" = \\"The following characters need special handling on some platforms: @ ? < & ' \\"\\";
+\\"Delivery.Widget.SpecialCharacters.COPY\\" = \\"The following characters need special handling on some platforms: @ ? < & ' \\". And then there is the percent-sign: most cases should be replaced (e.g. 99.9%% and %%1 and %%.1) but not the percentage-sign in strings like %1$@ or %2$d.\\";
```

Comment:
@johankool Here you can see how the `Localizable.strings` file will end up after this change. This is what you intended, right?